Svc/CmdDispatcher/CommandDispatcherComponentAi.xml

```diff
@@ -217,6 +217,22 @@
                 </arg>
            </args>       
         </event>
+        <event id="10" name="OpCodeReregistered" severity="DIAGNOSTIC" format_string = "Opcode 0x%04X is already registered to port %d slot %d" >
+            <comment>
+            Op code reregistered event
+            </comment>
+            <args>
+                <arg name="Opcode" type="U32">
+                    <comment>The opcode reregistered</comment>
+                </arg>          
+                <arg name="port" type="I32">
+                    <comment>The reregistration port</comment>
+                </arg>          
+                <arg name="slot" type="I32">
+                    <comment>The dispatch slot already allocated to the opcode</comment>
```

Comment:
The slot number number feels like an implementation detail to me, is this something we need to expose to users? Unless the entire opcode table mapping is exposed somewhere it doesn't seem like this value has any meaning.

I'd be tempted just to remove the dispatch slot from the EVR.

Reply:
I agree. Will remove the slot number from the EVR.